lab_1_keywords_tfidf/main.py

```diff
@@ -26,7 +26,7 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     Parameters:
     tokens (List[str]): Original token sequence
-    stop_words (List[str]: Tokens to exclude
+    stop_words (List[str]): Tokens to exclude
```

Comment:
only one? what about another?

Reply:
Actually, the rest seem to be correct. Of course, I'm not familiar with all the conventions of docstrings writing.